### ISSUE TITLE
Kord - saving patch somehow, is it possible?

### ISSUE THREAD

**Issue by radfaraf** [leandrob13/logue-hub#1]
Not an issue, but a question?

For a custom OSC like kord, is there a way to use software to save patches made for it, other writing down values by hand somewhere and then manually setting them again?

I program the patch settings via MIDI using a software controller such as this:
https://monosnap.com/file/gx8f2XAh7JeWLRjUyDzFVzgZE2cn5s

I notice you can kind of do if you don't use much of the special parameters, for example  Voic, Root, Scal, Sub, SHP, etc.. can't be recalled because you can't make a software knob for them and can only program them on the physical synth as far as I tell. Wondering if someone knows a hack for this to save those anyway somehow?

**Comment by leandrob13** [comment]
Hello! I looked into this today and I found out that there are MIDI NPRNs. The minilogue XD has them implemented for the OSC parameters but it is not the case for the NTS-1.

Logue-SDK issue thread: https://github.com/korginc/logue-sdk/issues/32

Reddit thread where they explain the implementation of NPRNs in the minilogue: https://www.reddit.com/r/LogueSDK/comments/rapqk4/nts1_midi_cc_for_custom_parameters/

This is one of everyone's main headaches when using custom oscillators with full configurable parameters. So far, it is not possible to set this up through MIDI.

I haven't found a safe way to persist state in the NTS-1. The EEPROM must be mapped in regions where the custom osc, rev, del and main firmware are stored so I don't want to risk and brick it. The bummer is that korg is not very responsive when dealing with issues reported in the logue-sdk github page.

**Comment by radfaraf** [comment]
Oh I happen to have the korg minilogue XD too got it recently, so I will try kord on that!

**Comment by radfaraf** [comment]
Yeah tried on the minilogue xd works much better because of the saving working there.
